src/CodeGen/ODataT4CodeGenerator.ttinclude

```diff
@@ -3473,12 +3473,35 @@ namespace <#= fullNamespace #>
                 global::System.Xml.XmlReader reader = CreateXmlReader(Edmx);
                 try
                 {
-                    return global::Microsoft.OData.Edm.Csdl.CsdlReader.Parse(reader, getReferencedModelFromMap);
-                }
-                finally

```

Comment:
where did the finally end up?

Reply:
Thanks for the catch. Finally, finally ended up back here.